tests/0120-asymmetric_subscription.c

```diff
@@ -170,9 +170,15 @@ int main_0120_asymmetric_subscription(int argc, char **argv) {
         rd_kafka_mock_topic_create(mcluster, "t4", _PART_CNT, 1);
 
 
-        do_test_asymmetric("roundrobin", bootstraps);
         do_test_asymmetric("range", bootstraps);
```

Comment:
This would be different for both the protocols as it is sticky in `consumer` but not in `classic`

Reply:
I can remove this tests too for consumer protocol as it's verifying the simple mock range implementation, that isn't sticky so it works in this case.